storybookjs/storybook
# Fixed a mistake in JSDoc

Closes #23119 



## What I did

I have corrected the incorrect code example in JSDoc, as noted in the issue above.



## How to test



1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Create setup file for testing Story with vitest etc.
3. Make sure there are no errors by writing according to JSDoc.

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
